### PR TITLE
Adding code to skip a dump with no files

### LINKED ISSUE #1761: Update IndexManager to filter next_dump with no dump_files
https://github.com/pulibrary/bibdata/blob/6c6934f352cecd316a7dcc7bf7b1abbc1f2812eb/app/models/index_manager.rb#L70

### PR DESCRIPTION
**Pull request by carolyncole**
fixes #1761

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -12,7 +12,8 @@ default: &default
 
 development:
   <<: *default
-  database: marc_liberation_dev
+  # database: marc_liberation_dev
+  database: marc_liberation_alma_dev
```
Comment (christinach) [inline config/database.yml:16]: @carolyncole is this change necessary

Comment (carolyncole) [reply config/database.yml]: @christinach no, thanks for catching that...  I removed it.

**Review by christinach — APPROVED** [review]
Thank you @carolyncole !
